lms/static/sass/elements-v2/_icons.scss

```diff
@@ -0,0 +1,31 @@
+.xseries-icon {
+  background: url('#{$static-path}/images/programs/xseries-icon.svg') no-repeat;
+}
+
+.micromasters-icon {
+  margin-top: $baseline * 0.05;
+  background: url('#{$static-path}/images/programs/micromasters-icon.svg') no-repeat;
+}
+
+.certificate-icon .certificate-body {
+  @include float(left);
+  @include margin-right($baseline*0.4);
+
+  margin-top: ($baseline/10);
+  width: 23px;
+  height: 20px;
+  padding: 2px;
+  background-color: $white;
+  border-style: solid;
+  border-width: 2px;
```

Comment:
I think you may be able to include the green and blue selectors at this level to make the relationship to the ```.certificate-icon``` class more obvious. It would be something like;
```
.green-certificate-icon& {
    fill: palette(success, accent);
    border-color: palette(success, accent);
}
```
You could also lose ```certificate``` from the class so it's just ```.green-icon```.